avogadro/qtgui/rwlayermanager.h

```diff
@@ -30,9 +30,10 @@ class AVOGADROQTGUI_EXPORT RWLayerManager : protected Core::LayerManager
   void addLayer(RWMolecule* rwmolecule);
   void setActiveLayer(size_t layer, RWMolecule* rwmolecule);
 
-protected:
   /** @return if @p layer is visible */
   bool visible(size_t layer) const;
+
+protected:
```

Comment:
No, you just need to know if the atom is visible, e.g. from your `closecontact.cpp` code:
```
  for (Index i = 0; i < molecule.atomCount(); ++i)
    isAtomEnabled[i] = m_layerManager.atomEnabled(i);
```